client/src/cordova/apple/xcode/ios/Outline/Classes/AppKitBridge/MainMenu.swift

```diff
@@ -0,0 +1,97 @@
+// Copyright 2025 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Cocoa
+
+class MainMenu: NSMenu {
+    private enum Messages {
+        static let quit = NSLocalizedString(
```

Comment:
I think you can use `String(localized:)` in Swift, which is more modern then NSLocalizedString: https://developer.apple.com/documentation/swift/string/localizationvalue

Up to you.